lib/common/environment.dart

```diff
@@ -5,10 +5,90 @@
 //  that can be found in the LICENSE file.
 //
 
+import 'dart:async';
+
+import 'package:autonomy_flutter/common/injector.dart';
 import 'package:autonomy_flutter/encrypt_env/secrets.dart';
+import 'package:autonomy_flutter/service/navigation_service.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:sentry/sentry.dart';
 
 class Environment {
+  static String _readKey(String key, String defaultValue,
+      {bool isSecret = false}) {
+    final res = (isSecret ? cachedSecretEnv : dotenv.env)[key];
+    if (res == null || res.isEmpty) {
+      unawaited(Sentry.captureMessage('Environment variable $key is not set'));
+      return defaultValue;
+    }
+    return res;
+  }
+
+  // check all the keys is set
+  static void checkAllKeys() {
+    final keys = [
```

Comment:
define constant for keys, secretKeys and missingKeys

Reply:
Fixed